### ISSUE TITLE
explain_prediction shouldn't limit feature count

### ISSUE THREAD

**Issue by kmike** [TeamHG-Memex/eli5#86]
Currently only top features are highlighted by default:

![limit](https://cloud.githubusercontent.com/assets/107893/20307024/a3d6f4d6-ab67-11e6-8bba-ed3a77d01bf1.png)

For me (23 more positive / 37 more negative) doesn't make much sense. Should we remove or increase limit for explain_prediction?

![nolimit](https://cloud.githubusercontent.com/assets/107893/20307097/0b6c53e8-ab68-11e6-8782-cfa6de1f145c.png)

**Comment by lopuhin** [comment]
Yes, I agree that it is confusing! Maybe we can make ``top=None`` (no limit, not sure it works yet) the default?

**Comment by kmike** [comment]
Hm, it may depend on data. For non-text data all features will be shown in a table, so limiting them could make sense. But no limit looks like a better default. I'll send a pull request.
